Makefile

```
@@ -120,6 +120,10 @@ test-for-release: envtest ## Run tests.
 test-integration: generate fmt vet envtest ## Run integration tests.
 	source ./tests/integration/env_vars.sh && $(GOTEST) ./tests/integration -v -race -run TestIstioJwt . && $(GOTEST) ./tests/integration -v -race -run TestOryJwt .
 
+.PHONY: test-upgrade
+test-upgrade: generate fmt vet envtest install ## Run API Gateway upgrade tests.
```

I think the envtest is a copy & paste leftover and can be removed from the godog integration tests
```suggestion
test-upgrade: generate fmt vet install ## Run API Gateway upgrade tests.
```